src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,37 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Pageblock Refresh
+Pageblocks have a refresh option to automatically renew the pageblock data without having to reload the entire page. 
+This is useful for pageblocks that display data that changes frequently, such as a list of recent records, an update to a chart, or a list of recent comments.
+
+=== Pageblocks With Refresh
+Not all pageblocks have the refresh option enabled. Below is a list of pageblocks that have the refresh option enabled.
+
+* Record List
+* Chart
+* Comment
+* Iframe
+* Metric
+* Progress
+* Record Organizer
+* Report
+* Social Feed
+* Calendar
+
+=== Enabling Pageblock Refresh
+To enable refreshing on a pageblock, you will need to find the refresh option in the *general tab* when you edit the pageblock. 
+image:compose-configuration/page-block-general.png[Pageblock General Tab, width=49%]
+The refresh option is divided into auto-refresh and manual refresh.
+
+==== Auto-refresh
+When set, pageblocks that have this configuration will automatically refresh based on a set interval. To activate auto refresh, you'll want to enter a numerical value into the input above the *Show refresh button* toggle. The lowest interval allowed is 5 seconds. Setting an interval lower than that would make the input default to 5 seconds. To turn auto-refresh off on a block, you only need to set its value to zero.
+
+==== Manual Refresh
+To activate, toggle the *Show refresh button* switch to its on state. When set, pageblocks that have this configuration will have a refresh button in the top right corner of the pageblock. This button will allow the user to manually refresh the pageblock. To turn manual refresh off on a block, you only need to set its value to false, i.e toggle the switch off.
```

Comment:
`This button will allow the user to manually refresh the pageblock`

This bit sounds redundant to me. It might be worth keeping it to make it a bit longer (which can be both good or bad) but I'd consider either removing it or merging it with the previous sentence.